.github/workflows/publish-releasenotes-outside.yml

```diff
@@ -31,5 +31,6 @@ jobs:
       - name: Run Shell Script
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PAT: ${{ secrets.PAT }}
```

Comment:
Why it matters? It could be be bot

Reply:
Oh okay, I though you wanted it to be your name since that used to be the case before. Reverted. Done.
Here is release from github-actions: https://github.com/stoyanK7/checkstyle/releases/tag/checkstyle-10.8.0
Actions: https://github.com/stoyanK7/checkstyle/actions/runs/4270968478